schedule/schedule-appengine/src/main/client/jasify/admin/multipass/admin-multipass.js

```diff
@@ -20,7 +20,7 @@
         vm.deleteRule = deleteRule;
         vm.init = init;
         vm.activityTypes = [];
-        vm.multipass = multipass;
+        vm.multipass = multipass.data !== undefined ? multipass.data : {};
```

Comment:
I like using the `angular.isXX` methods ([see doc](https://code.angularjs.org/1.3.19/docs/api/ng/function)), like:

~~~javascript
vm.multipass = angular.isObject(multipass.data) ? multipass.data : {};
~~~

So I am sure it is alway an Object `{...}`  or whenever I want an Array `[...]`